src/components/common/Banner/ActiveBannerSlot.tsx

```
@@ -3,15 +3,16 @@ import { FC } from 'react';
 
 import AdsBanner from '@/components/common/Banner/AdsBanner';
 import { ClosingBanner } from '@/components/common/Banner/ClosingBanner';
-
+import WelcomeBannerContainer from '@/components/common/Banner/WelcomeBanner/WelcomeBannerContainer';
 interface ActiveBannerSlotProps {}
 
 const ActiveBannerSlot: FC<ActiveBannerSlotProps> = ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)